src/utils/time.js

```diff
@@ -65,31 +65,34 @@ export const getCurrentYear = () => new Date().getFullYear();
 
 /**
  * Returns the current month
- * @returns {String} Current month
+ * @returns {Number} Current month
  */
-export const getCurrentMonth = () => padWithZeroes(new Date().getMonth() + 1);
+export const getCurrentMonth = () => new Date().getMonth() + 1;
 
 /**
- * Returns the last month
+ * Returns the last available month
+ * @param {Date} date Current date
  * @param {Number} lagTime Delay in days (10 days for ERA5-Land)
- * @returns {String} Last month
+ * @returns {Array} Last year and month
  */
-export const getLastMonth = (lagTime = 10) => {
-  const currentMonth = getCurrentMonth();
-  const currentDay = new Date().getDate();
+export const getLastMonth = (date, lagTime = 10) => {
+  // Based on https://stackoverflow.com/a/7937257
+  const newDate = new Date(date.getTime());
+  const month = newDate.getMonth();
+  const monthsBack = newDate.getDate() > lagTime ? 1 : 2;
```

Comment:
Maybe this should be >= to also include the day of the month indicated by lagTime?

Reply:
I changed the name from lagTime to lagDays to make it more clear and kept > (above) check.